lib/open_project/access_control.rb

```diff
@@ -70,7 +70,7 @@ def permission(action)
 
           permissions.detect { |p| p.controller_actions.include?(permission_path) }
         else
-          permissions.detect { |p| p.name == action }
+          permissions.detect { |p| p.name == action&.to_sym }
```

Comment:
This change is a bit unexpected, is this tested anywhere?

Reply:
I'll put it in the other place where this is needed, and not in the general class 👍